.azure_pipelines/job_templates/olive-build-template.yaml

```diff
@@ -99,6 +99,12 @@ jobs:
       alertWarningLevel: 'High'
     displayName: Component Detection
 
+  - task: CredScan@3
```

Comment:
not a big issue but can we add the cred scan job under the test_olive job above? instead of test_examples job?

Reply:
Moved after test_olive job